include/geode/basic/attribute.h

```diff
@@ -370,6 +375,11 @@ namespace geode
             set_value( to_element, interpolation.compute_value( *this ) );
         }
 
+        void set_to_interpolable()
```

Comment:
Open question : 
void set_properties( AttributeProperties ) ?

Reply:
Instead of `change_properties`